src/main/java/com/epam/javacourses2016/task11/SolverTask11.java

```diff
@@ -16,16 +16,26 @@
      * @param peoples Список с именами участников.
      * @return Имя последнего оставшегося.
      */
-    public String emulate(ArrayList<String> peoples) {
```

Comment:
Don't you feel confused about copy-paste in those methods?)

Realization for ArrayList is not effective